initializr-generator/src/test/java/io/spring/initializr/generator/buildsystem/maven/MavenBuildWriterTests.java

```diff
@@ -587,4 +548,11 @@ private void generatePom(MavenBuild mavenBuild, Consumer<NodeAssert> consumer)
 		consumer.accept(new NodeAssert(out.toString()));
 	}
 
+	private MavenBuild demoMavenBuild() {
```

Comment:
We prefer to keep commits as focussed as possible. While certainly not a bad change to make, this is unrelated to adding support for settings `<extensions>true</extensions>`. Could you revert this part of the changes please?

Reply:
Ok. I've just stumbled on it and realised I will be repeating the exact same code, so thought would extract it along on the fly. Will revert it.